bin/version-include.sh

```diff
@@ -63,50 +64,82 @@ function deployV4MInfo() {
   helm upgrade --install \
     -n "$NS" \
     --values $valuesYAML \
-    v4m ./v4m-chart
+    $releaseName ./v4m-chart
+
+  getHelmReleaseVersion "$NS" "$releaseName"
 }
 
 function removeV4MInfo() {
   NS=$1
+  releaseName=$2
   if [ -z "$NS" ]; then
```

Comment:
If the purpose of this function is to remove the v4m Helm release, why is releaseName a parameter?  Wouldn't it ALWAYS be set to "v4m"?

Reply:
We have multiple releases - v4m, v4m-viya, and v4m-tenant-[tenant] - The deploy and remove functions used to be hardcoded to just 'v4m' since that's all we had. Now they're generic enough to handle all 3 situations.